doc/user/content/sql/values.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "VALUES"
+description: "`VALUES` constructs a table from value expressions."
+menu:
+  main:
+    parent: commands
+---
+
+The `VALUES` expression constructs a table from value expressions.
+
```

Comment:
I would rather not use this language, since Materialize uses `TABLE` to mean something different. I'm assuming "table" is a more abstract SQL concept that means something like what I usually call "relation", but I think we should disambiguate here.